horreum-web/src/domain/user/ApiKeys.tsx

```diff
@@ -110,7 +110,7 @@ export default function ApiKeys() {
 
     const performRenameApiKey = () => {
         if (renameKeyId) {
```

Comment:
Instead of adding `!` that could fail bad if that is actually undefined, I would rather add the check in the if
```suggestion
        if (renameKeyId && renameKeyName) {
```